sportsref/ncaaf/players.py

```diff
@@ -147,8 +152,17 @@ def rushing_and_receiving(self):
         :returns: Pandas DataFrame with stats.
         """
         doc = self.getDoc()
-        table = doc('#rushing')
+        table = doc('#all_rushing')
```

Comment:
Those lines parse the comment from the 'all_rushing' table which is the only way I could get at that data. It's pretty hacky, but you are welcome to try the other way. I got no results.